src/test/java/org/apache/maven/shared/filtering/DefaultMavenResourcesFilteringTest.java

```diff
@@ -758,20 +752,20 @@ private void createTestDataStructure( File sourceDirectory )
         File dir1 = new File( sourceDirectory, "dir1" );
 
         dir1.mkdirs();
-        FileUtils.fileWrite( new File( dir1, "foo.txt" ), "UTF-8", "This is a Test File" );
+        FileUtils.write( new File( dir1, "foo.txt" ), "This is a Test File", "UTF-8" );
 
         File emptyDirectory = new File( sourceDirectory, "empty-directory" );
         emptyDirectory.mkdirs();
 
-        FileUtils.fileWrite( new File( emptyDirectory, ".gitignore" ), "UTF-8", "# .gitignore file" );
+        FileUtils.write( new File( emptyDirectory, ".gitignore" ), "# .gitignore file", "UTF-8" );
 
         File emptyDirectoryChild = new File( sourceDirectory, "empty-directory-child" );
         emptyDirectory.mkdirs();
 
         File emptyDirectoryChildEmptyChild = new File( emptyDirectoryChild, "empty-child" );
         emptyDirectoryChildEmptyChild.mkdirs();
 
-        FileUtils.fileWrite( new File( emptyDirectoryChildEmptyChild, ".gitignore" ), "UTF-8", "# .gitignore file" );
+        FileUtils.write( new File( emptyDirectoryChildEmptyChild, ".gitignore" ), "# .gitignore file", "UTF-8" );
```

Comment:
simply using Files.write could avoid external dependencies

Reply:
not quite a 1:1 replacement. What we really need is Files.writeString from Java 11. We'd also need to change the File objects to Path objects.   I'll leave this change for a separate PR if someone wants to take it.